integration_tests/commands/hincrby_test.go

```diff
@@ -0,0 +1,46 @@
+package commands
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHINCRBY(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	testcases := []TestCase{
+		{
```

Comment:
Would it be okay to add a test case when incrementing with two big yet 64 bit integers would overflow. This will help Dice to check every scenario.
nit: name field in test cases help future contributors to understand about the test cases.

Reply:
Yes please add all edge cases which you can think of. Don't hesitate, the more the number, the more we are safe.